code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -654,6 +654,37 @@
 /obj/item/weapon/storage/fancy/food_box/chicken_bucket/update_icon(var/itemremoved = 0)
 	return
 
+
+/obj/item/weapon/storage/fancy/food_box/vox_chicken_bucket
+	name = "vox chicken bucket"
+	desc = "I’ll have two number 9s, a number 9 large, a number 6 with extra dip, a number 7, two number 45s, one with cheese, and a large soda."
+	icon_state = "vox_drumstick_bucket"
+	item_state = "kfc_bucket"
+	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/food.dmi', "right_hand" = 'icons/mob/in-hand/right/food.dmi')
+	icon_type = "drumstick"
+	can_only_hold = list("/obj/item/weapon/reagent_containers/food/snacks/vox_chicken_drumstick")
+	starting_materials = list(MAT_CARDBOARD = 3750)
+	w_type=RECYK_MISC
+
+/obj/item/weapon/storage/fancy/food_box/vox_chicken_bucket/New()
+	..()
+	for(var/i=1; i <= storage_slots; i++)
+		new /obj/item/weapon/reagent_containers/food/snacks/vox_chicken_drumstick(src)
+	return
```

Comment:
return at the end of the proc does nothing, remove it

Reply:
Same as above.

Reply:
don't care

Reply:
done